File: angular/src/app/shared/service/orderdetail/models/orderdetail-dto.ts
import { EntityDto } from '@abp/ng.core';

export class OrderDetailDto extends EntityDto<string>{
  orderId: string;
  price: number;
  quantity: number;
  orderName: string;
  customerId: string
  imagesource: string;
  size:string;

  constructor(initial: Partial<OrderDetailDto> = {}) {
    super(initial);
  }
}

File: aspnet-core/src/Selling_Website.Application.Contracts/OrderDetails/Dto/OrderDetailDto.cs
using System;
using Volo.Abp.Application.Dtos;

namespace Selling_Website.OrderDetails.Dto
{
    public class OrderDetailDto : FullAuditedEntityDto<Guid>
    {
        public Guid? OrderId { get; set; }
        public int? Price { get; set; }
        public string Quantity { get; set; }
        public string OrderName { get; set; }
        public Guid? CustomerId { get; set; }
        public string Imagesource { get; set; }
    }
}
